test/controllers/ChooseAppealReasonController.test.ts

```diff
@@ -0,0 +1,62 @@
+import { expect } from 'chai';
+import request from 'supertest';
+import { createApp } from '../ApplicationFactory';
+
+import { Appeal } from 'app/models/Appeal';
+import { ApplicationData } from 'app/models/ApplicationData';
+import { CHOOSE_REASON_PAGE_URI } from 'app/utils/Paths';
+
+describe('ChooseAppealReasonController', () => {
+
+    let env: NodeJS.ProcessEnv;
+
+    before( () => {
+        env = process.env;
```

Comment:
That is becoming bit tedious as few tests already have this 1) capture envs 2) modify envs 3) restore envs approach. Can you propose something better? Maybe all future flags should be enabled by default for tests and only in rare instances value should be modified in the test?

Reply:
How do I modify the default values for test environment variables such that they aren't dependent on the runtime environment variables? My understanding is that the test environment should be consistent and independent from runtime assumptions.

Reply:
Resolved: Environment manipulation is no longer required due to change in feature toggle approach.